FreeRTOS/iot-reference-esp32c3
# Fix Demo Build Errors

Description
-----------

Shrink the idle task stack and the timer callback stack by 512, and reduce the number of OTA buffers held to fit into minimal size when using ESP IDF v5.1

Test Steps
-----------

Shrank sizes using `idf.py menuconfig`, copied relevant config options to the default config.

Checklist:
----------



Related Issue
-----------

#49 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
